articles/logic-apps/connect-virtual-network-vnet-isolated-environment.md

```diff
@@ -49,6 +49,8 @@ This article shows how to complete these tasks:
   * Your virtual network needs to have four *empty* subnets for creating and deploying resources in your ISE. You can create these subnets in advance, or you can wait until you create your ISE where you can create subnets at the same time. Learn more about [subnet requirements](#create-subnet).
 
   * Subnet names need to start with either an alphabetic character or an underscore and can't use these characters: `<`, `>`, `%`, `&`, `\\`, `?`, `/`. 
+  
+  * When deploying the ISE through ARM, first ensure one of these empty subnets is delegated to **Microsoft.Logic/integrationServiceEnvironment**.  This is not required when deploying through the Azure Portal.
```

Comment:
Based on the reasons below, here's a revised version:

"If you want to deploy the ISE through an Azure Resource Manager template, first make sure that you delegate one empty subnet to `Microsoft.Logic/integrationServiceEnvironment`. You don't need this delegation when you deploy through the Azure portal."

- ARM is not an approved acronym for Azure Resource Manager, so please always spell out or use "Resource Manager" on second mention. See ["Azure Resource Manager" in the Microsoft Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=39369).
- Bold formatting is generally reserved for UI elements. Is "Microsoft.Logic/integrationServiceEnvironment" a UI element in this context? For Resource Manager templates, use code formatting when you refer to JSON elements.
- "This is..." is ambiguous for localization and grammatical reasons, so always use a noun after "This", for example, "This delegation..."
- Please remove extra single space before "This".
- "portal" is lowercase.
- Use active voice when possible.